consensus/e2e_tests/pacemaker_test.go

```diff
@@ -172,6 +172,118 @@ func forcePacemakerTimeout(t *testing.T, clockMock *clock.Mock, paceMakerTimeout
 	advanceTime(t, clockMock, paceMakerTimeout+10*time.Millisecond)
 }
 
+func TestPacemaker_MinBlockTime(t *testing.T) {
```

Comment:
```suggestion
func TestPacemaker_MinBlockTime_???(t *testing.T) {
```

Happy Path? Sad Path? Need a suffix